instances/unitOfMeasurement/kilovolt.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/unitOfMeasurement/kilovolt",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UnitOfMeasurement",
+  "definition": "One thousand volts.",
```

Comment:
please match the definition form you used for hertz/kilohertz (here naturally vold / kilovolt)